tests/vagrant/Vagrantfile

```diff
@@ -6,15 +6,15 @@ VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
-	config.vm.box = "ubuntu/trusty64"
+	config.vm.box = "ubuntu/xenial64"
 	config.vm.box_check_update = false
 
 	config.ssh.forward_agent = true
+	config.vm.network :private_network, ip: '192.168.10.10'
```

Comment:
Hmmmm you must have something that is trying to share a folder via NFS, have you got anything in ~/.vagrant.d/Vagrantfile ? Either that or for some reason Vagrant is trying to mount the default share via NFS, which it shouldn't do.